modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,97 @@
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature only.
+====
+
+* With this update, users can configuration notifications in a dedicated `NotificationsConfiguration` custom resource (CR). This process replaces the workflow in earlier releases of {gitops-title}. To configure this feature in earlier releases, you had to edit the `argocd-notifications-cm` ConfigMap manually. Users can now set Triggers, Templates, Services, and Subscriptions as fields in the new `NotificationsConfiguration` custom resource (CR), which is managed by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
+
+* With this update, two new fields `.spec.applicationSet.sourceNamespaces` and `.spec.applicationSet.scmProviders` are introduced in ArgoCD CRD to support `ApplicationSet` in any namespaces. Administrators use it to define certain namespaces that manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3754[GITOPS-3754]
+
+* With this update, Argo CD server has the required permissions to manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3762[GITOPS-3762]
+
+* With this update, you can configure log levels such as `debug`, `info`, `warn`, `error`, `panic` and `fatal`. The default log level set for the output is `info`. Users can select an appropriate log level by adding the environment variable `LOG_LEVEL` in Red Hat OpenShift GitOps subscription's field, `.spec.config.env`. link:https://issues.redhat.com/browse/GITOPS-4016[GITOPS-4016]
+
+.Example
+[source,yaml]
+----
+   yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  ...
+  config:
+    env:
+    - name: LOG_LEVEL
+      value: "error"
+----
+
+* With this feature, Argo CD uses the wildcard values in `sourceNamespaces` to specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the Argo CD YAML with `.spec.sourceNamespaces`. link:https://issues.redhat.com/browse/GITOPS-3935[GITOPS-3935]
+
+.Example
+[source,yaml]
+----
+yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-wildcard-pattern
+spec:
+  sourceNamespaces:
+    - app-team-*
+   - namespace-2
+----
+
+In this example, permissions are granted to namespaces that match the pattern `app-team-*`, such as `app-team-1`, `app-team-2`, and namespace-2, which does not use wildcard values.
+
+To grant permissions for all the namespaces on the Argo CD cluster using the `*` wildcard pattern, the following Argo CD CR configuration must be used:
+
+.Example
+[source,yaml]
+----
+yaml
```

Comment:
do we need this line?? We usually start from the `apiVersion` field.

Reply:
Fixed.